server/src/com/openrsc/server/net/rsc/ActionSender.java

```diff
@@ -807,6 +808,7 @@ private static Packet prepareServerConfigs(Server server) {
 		configs.add((byte) (server.getConfig().WANT_CERT_AS_NOTES ? 1 : 0)); // 79
 		configs.add((byte) (server.getConfig().WANT_OPENPK_POINTS ? 1 : 0)); // 80
 		configs.add((byte) (server.getConfig().OPENPK_POINTS_TO_GP_RATIO)); // 81
+		configs.add((byte) (server.getConfig().SHOW_TUTORIAL_SKIP_OPTION ? 1 : 0)); // 82
```

Comment:
not needed once Client_Base file is reverted

Reply:
@Hubcapp This has been removed.